### PR TITLE
Remove Region class from the scrape module and update all the server …

### PR DESCRIPTION
**Pull request by skywhale**

### CODE REVIEW

File: tests/server_test_cases/config_tests.py [hunk 1]
```diff
@@ -67,9 +67,9 @@ def tearDown(self):
         self.go('/haiti?lang=en&flush=config')
 
     def get_admin_page_error_message(self):
-        error_div = self.s.doc.first('div', class_='error')
+        error_div = self.s.doc.cssselect('div.error')
```
Comment (gimite) [inline tests/server_test_cases/config_tests.py]: Nit: May be better to call it error_divs.

**Comment by skywhale** [comment]
I mistakenly used a wrong email address for committing the changes. I pushed updated refs for fix, but if it doesn't fix the warning, I'm just going to sign the contract with the new email address.

**Comment by gimite** [comment]
Great job, thanks! LGTM. Just one minor comment.

**Comment by skywhale** [comment]
Thanks. Merged :)

**Pull request merged**